libs/ha/ha_common_libs_k8s.py

```diff
@@ -1256,7 +1256,9 @@ def setup_mock_monitor(node_obj):
         return True
 
     def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_type,
-                                  node_type='data', resource_cnt=1, node_cnt=1, delay=None):
+                              node_type='data', resource_cnt=1, node_cnt=1, delay=None,
+                              specific_info=False, generation_id=None, node_id=None,
```

Comment:
specific_info is expected to be of dict type, not bool. Its default value should be None.

Reply:
Changed